src/ray/core_worker/experimental_mutable_object_provider.cc

```diff
@@ -33,6 +33,10 @@ MutableObjectProvider::~MutableObjectProvider() {
   }
   RAY_CHECK(object_manager_->SetErrorAll().code() == StatusCode::OK);
 
+  for (const auto &io_context : io_contexts_) {
```

Comment:
@rkooo567 I am not familiar with MutableObjectProvider. Looks like it dynamically creates thread + io contexts? In this PR I added a lag probe to `instrumented_io_context`s so even if here we cleaned all io_work, we still need to stop the context to remove the infinitely creating lag probes. Please LMK if this is not wanted - we can opt-out of lag probes in this mutable object provider.

Also - why do we dynamically create io_contexts here? Skipping through the code I feel 1 thread + 1 io_context can do the trick.

Reply:
disabled lag probes for io ctxs created by this class.